src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -334,15 +334,31 @@ export const TimelineNav = ({
   };
 
   /**
-   * onClick function
-   * 1) Triggered by 'Back' button on < lg viewports
-   * 2) Return focus to the button that was clicked to open the current panel; activeIndexState holds the index of the last nav item selected in the timelinNavItemRefs array
-   * 3) Body panel is visible/hidden depending on true/false value of isActive; hide it by setting isActive to false
+   * onClick (used by 'Back' button on < lg viewports)
+   * 1) Return focus to the button that was clicked to open the current panel; activeIndexState holds the index of the last nav item selected in the timelinNavItemRefs array
+   * 2) Body panel is visible/hidden depending on true/false value of isActive; hide it by setting isActive to false
    */
   const onClick = () => {
-    timelineNavItemRefs[activeIndexState].current?.focus(); /* 2 */
-    setIsActive(false); /* 3 */
+    timelineNavItemRefs[activeIndexState].current?.focus(); /* 1 */
+    setIsActive(false); /* 2 */
   };
+
+  /**
+   * onFocus (used by 'Back' button on <lg viewports)
```

Comment:
Can you give it a more specific name so it's more obvious that this isn't intended to be used when anything is focused, only when the back button is focused in mobile view?